src/test/java/tests/AddClientProceedsPageTest.java

```diff
@@ -0,0 +1,203 @@
+package tests;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import pages.*;
+
+public class AddClientProceedsPageTest extends BasePage {
+
+    String addNewPaymentWindow = "https://ps.contapp.ro/venituri/factura/creaza?preview=false";
```

Comment:
te rog sa faci pull si iti vei pune link de verificare in Enum ul creat de Mihai pe repository

Reply:
Rezolvat